### DISCUSSION TITLE
V177 lost fan settings after update

### DISCUSSION THREAD

**Discussion by R3d2q** [Rem0o/FanControl.Releases#2237] [General]
Again losst all fan settings and names.
After update was everything fine. After computer returned from hibernation Fan Control could't detect fans as sometimes happened. After restarting FanControl lost all fan names and settings. Could't restore from latest versions configurations. Tried older version configurators and got names back without curve configurators. There was only manual configuraton adjusted.

**Comment by Rem0o** [comment]
Shouldn't lose anything.   Lookup in your configuration folder, your configs should be there.   If a config was migrated, there should be a backup folder.

**Comment by jamesoloughlin** [comment]
Backup configs do not import after update and loss. Says "There is no fan curves or custom sensors to import".

**Comment by Beat-g** [comment]
same here, i lost all my settings after updating from 178 to 181 and my config file was not move to the backup folder
